playwright.config.ts

```diff
@@ -38,9 +38,14 @@ export default defineConfig({
 		...(DEV && { headless: false })
 	},
 	projects: [
+		/* Test against desktop browsers. */
 		{
 			name: 'Google Chrome',
-			use: { ...devices['Desktop Chrome'], channel: 'chrome' }
+			use: { ...devices['Desktop Chrome'] }
```

Comment:
honest doubt: is it ok to remove the channel? what is the default?

Reply:
The channel is only relevant in Chrome. Basically when there is no channel playwright will use the latest Chromium browser. If there is a channel it uses a specific version. I don't think its necessary to test a specific version of chrome, so we can also go with latest Chromium.

Reply:
Asking because it seems this change changed all the screenshots that we already had for Chrome: you can notice that they are quite different (scrollbar, less viewport but same size, strange)